repository_service_tuf_api/metadata.py

```diff
@@ -75,3 +76,129 @@ def post_metadata(payload: MetadataPostPayload) -> MetadataPostResponse:
     return MetadataPostResponse(
         data={"task_id": task_id}, message="Metadata update accepted."
     )
+
+
+class SigningData(BaseModel):
+    metadata: Dict[Roles, TUFMetadata]
+
+
+class MetadataSignGetResponse(BaseModel):
+    data: Optional[SigningData]
```

Comment:
Why `data` is optional?
Even if there is no metadata for signing we return, right?

Reply:
The response can be an HTTP 500, 404, 400, etc., without data or a body.